index.js

```diff
@@ -10,12 +10,24 @@ const chalk = require('chalk')
  * @param {Object} creds AWS credentials
  */
 const createAndTagTable = async function createAndTagTable(region, tableName, setting, tags, cli, creds) {
-  cli.consoleLog(`CreateGlobalTable: ${chalk.yellow(`Creating new table ${tableName} in ${region} region...`)}`)
   const dynamodb = new AWS.DynamoDB({
     credentials: creds,
     region,
   })
   try {
+
+    try {
+      await dynamodb.describeTable({TableName: tableName}).promise();
+      cli.consoleLog(`CreateGlobalTable: ${chalk.yellow('Backup region table already exists in ${region}. Skipping creation...')}`);
```

Comment:
@johndww  for global tables, the destination table should be empty as well, can you please add that check so that final association doesn't fail and that condition is handled gracefully
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/globaltables_reqs_bestpractices.html